numpy/distutils/tests/test_system_info.py

```diff
@@ -0,0 +1,208 @@
+from __future__ import division, absolute_import, print_function
+
+import os
+from tempfile import mkstemp, mkdtemp
+
+import distutils
+from numpy.testing import *
```

Comment:
Can you import here only what's needed? Looks like ``TestCase`` and ``run_module_suite`` only now, but note that in general we prefer to use ``assert_`` functions from ``numpy.testing`` rather than ``nose`` functions.